### PR TITLE
Pimp my firetruck

### PR DESCRIPTION
**Pull request by luistorres**
* Adds danke styles to the display
* Adds api to set firefighter busy
* Adds real seeds list
* Adds logic to make state transition cyclic inactive -> active -> busy 
-> inactive

### CODE REVIEW

File: src/frontend/components/Firefighter/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        onClick={this.handleFirefighterCheckChange}
+      >
+        <div styleName="block name">{name}</div>
+        <div styleName="block status">{label[status]}</div>
+      </button>
+    );
+  }
+}
+
+Firefighter.propTypes = {
```
Comment (jfranciscosousa) [inline src/frontend/components/Firefighter/index.js:52]: Dont like this syntax, can you use `static propTypes`?

File: src/frontend/components/Firefighter/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  status: PropTypes.string.isRequired,
+  addActiveFirefighter: PropTypes.func.isRequired,
+  removeActiveFirefighter: PropTypes.func.isRequired,
+  addBusyFirefighter: PropTypes.func.isRequired
+};
+
+export default Firefighter;
```
Comment (jfranciscosousa) [inline src/frontend/components/Firefighter/index.js:61]: Export default immediatly at the top so people don't need to scroll down to see this export.

File: src/frontend/utils/api.js [hunk 3]
```diff
@@ -8,5 +8,9 @@ export const getFirefighters = () => axios.get(`${API_URL}/api/firefighters`);
 export const addActiveFirefighter = id =>
   axios.put(`${API_URL}/api/firefighters/${id}`, { status: "active" });
 
-export const removeActiveFirefighter = id =>
-  axios.put(`${API_URL}/api/firefighters/${id}`, { status: "inactive" });
+const updateFirefighter = (id, state) =>
+  axios.put(`${API_URL}/api/firefighters/${id}`, { status: state });
+
+export const removeActiveFirefighter = id => updateFirefighter(id, "inactive");
```
Comment (jfranciscosousa) [inline src/frontend/utils/api.js:14]: Do you need this one?

File: src/frontend/components/Firefighter/index.js (hunk 1 above)
Comment (luistorres) [reply src/frontend/components/Firefighter/index.js:52]: As you're saying, this comes down to preferences.
We are using this in the new projects, and I kinda understand it: you leave inside the class what really matters.
I like it and  it's not an issue so leave it 🙃

File: src/frontend/components/Firefighter/index.js (hunk 2 above)
Comment (luistorres) [reply src/frontend/components/Firefighter/index.js:61]: Again, that's not a "thing". You have many patterns, you can follow the one you like. This one comes from the react pure class snippets, which is what I am using atm.

File: src/frontend/utils/api.js (hunk 3 above)
Comment (luistorres) [reply src/frontend/utils/api.js:14]: Yes. We have 3 states: active, busy and inactive.

**Review by jfranciscosousa — APPROVED** [review]
👍

**Pull request merged**